File: Modul 4/Soal 1/PRAK401-2410817210019-AchmadReihanAlfaiz.py
number_input, symbol_input = input().split()
number_input = int(number_input)

min_value, max_value = 1, 50

for i in range(min_value, max_value + 1):
    if i % number_input == 0:
        print(symbol_input, end=" ")
        continue

    print(i, end=" ")

File: Modul 4/Soal 1/PRAK401-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int numberInput, symbolInput;
     scanf("%d %c", &numberInput, &symbolInput);

     int min = 1, max = 50;

     for (int i = min; i <= max; i++)
     {
          if (i % numberInput == 0)
          {
               printf("%c ", symbolInput);
               continue;
          }

          printf("%d ", i);
     }
     return 0;
}